server/routes/routerRequestHandlers.test.ts

```diff
@@ -451,6 +451,34 @@ describe('routerRequestHandlers', () => {
       expect(next).toHaveBeenCalled()
     })
 
+    it('should return prisoners first name and last name without whitespace and a capital letter at the start', async () => {
```

Comment:
Oh yeah; now we're talking! This is perfect. Loving the test for it 👍

Reply:
Testing... I'm getting there 😆